serverless/src/getContactInfo/__tests__/getCmrPreferencesData.test.js

```diff
@@ -15,28 +14,26 @@ beforeEach(() => {
 describe('getCmrPreferencesData', () => {
   test('correctly requests a users data from cmr-ordering', async () => {
     jest.spyOn(getEarthdataConfig, 'getEarthdataConfig').mockImplementation(() => ({ cmrHost: 'https://cmr.com' }))
-
-    jest.spyOn(getClientId, 'getClientId').mockImplementation(() => ({ background: 'eed-edsc-test-serverless-background' }))
+    jest.spyOn(getClientId, 'getClientId').mockImplementation(() => ({ lambda: 'eed-edsc-test-serverless-lambda' }))
 
     nock(/cmr/)
       .matchHeader('Authorization', 'Bearer fake.access.token')
-      .matchHeader('Client-Id', 'eed-edsc-test-serverless-background')
+      .matchHeader('Client-Id', 'eed-edsc-test-serverless-lambda')
       .matchHeader('X-Request-Id', 'mock-request-id')
       .post(/ordering\/api/)
       .reply(200, {
-        data: {
-          user: {
-            ursId: 'urs_user',
-            notificationLevel: 'INFO'
-          }
+        user: {
+          mock: 'cmr'
         }
       })
 
-    const cmrData = await getCmrPreferencesData('urs_user', 'fake.access.token')
+    const cmrData = await getCmrPreferencesData('urs_user', 'fake.access.token', 'test')
 
-    expect(cmrData).toEqual({
-      ursId: 'urs_user',
-      notificationLevel: 'INFO'
+    const { data: preferences } = cmrData
```

Comment:
I really like this.  Data is ambiguous, but calling it preferences helps clarify what we are looking at :-)